src/centrality/truss.cpp

```diff
@@ -0,0 +1,271 @@
+/*
+
+  Copyright 2017 The Johns Hopkins University Applied Physics Laboratory LLC. All Rights Reserved.
```

Comment:
All rights aren't really reserved ;-) Quite a few rights are granted perpetually through the GPL.